### PR TITLE
kudo: add KUDO v.0.9.0

### PR DESCRIPTION
**Pull request by zmalik**
Adding the KUDO `v0.9.0` as this is the most recent release of KUDO

### CODE REVIEW

File: addons/kudo/0.9.x/kudo-1.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kudo
+  labels:
+    kubeaddons.mesosphere.io/name: kudo
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.9.0"
```
Comment (shaneutt) [inline addons/kudo/0.9.x/kudo-1.yaml]: Revisions need to have a `-${revision_number}` for the current version:

```suggestion
    catalog.kubeaddons.mesosphere.io/addon-revision: "0.9.0-1"
```

**Comment by nfnt** [comment]
I'm not sure what is being tested in the CI, but the KUDO driver in konvoy is currently using built with KUDO 0.8.0 and might not work with an updated KUDO. This isn't something we currently test in KUDO, though we're currently working on this.

**Comment by zmalik** [comment]
@nfnt what do you mean with `might not work with an updated KUDO` I have tested `0.9.0` in konvoy. What is different for addons use case?

**Comment by nfnt** [comment]
Great, if it's working with the existing driver then it's fine!

**Comment by zmalik** [comment]
I think you mean to test via kudo driver integration right?
we will need to test that manually and might not cut it for upcoming konvoy release

**Comment by zmalik** [comment]
tested 
```
 k apply -f kudo.yaml
clusteraddon.kubeaddons.mesosphere.io/kudo created

~/repos/kudo-testing ⌚ 15:16:12
$ (⎈ kudo-testing-admin@kudo-testing:default) k get clusteraddons
NAME                             READY   STAGE
awsebscsiprovisioner             true    deployed
cert-manager                     true    deployed
dashboard                        true    deployed
defaultstorageclass-protection   true    deployed
external-dns                     true    deployed
kommander                        true    deployed
kudo                             true    deployed
traefik                          true    deployed
velero                           true    deployed

~/repos/kudo-testing ⌚ 15:16:22
$ (⎈ kudo-testing-admin@kudo-testing:default) k get pods -n kudo-system
NAME                        READY   STATUS    RESTARTS   AGE
kudo-controller-manager-0   1/1     Running   0          16s

~/repos/kudo-testing ⌚ 15:16:30
$ (⎈ kudo-testing-admin@kudo-testing:default) k logs kudo-controller-manager-0 -n kudo-system
2019/12/19 14:16:17 KUDO Version: version.Info{GitVersion:"0.9.0", GitCommit:"9981ed2b", BuildDate:"2019-12-17T17:27:45Z", GoVersion:"go1.13", Compiler:"gc", Platform:"linux/amd64"}
2019/12/19 14:16:17 setting up manager
2019/12/19 14:16:18 Registering Components
2019/12/19 14:16:18 setting up scheme
2019/12/19 14:16:18 Setting up operator controller
2019/12/19 14:16:18 Setting up operator version controller
2019/12/19 14:16:18 Setting up instance controller
2019/12/19 14:16:18 Starting the Cmd
```

**Comment by zmalik** [comment]
let's bump to version `0.1.4` as `0.1.3` has a missing subresource status for instance CRD

**Comment by zmalik** [comment]
https://github.com/mesosphere/charts/pull/322 is merged now

**Comment by zmalik** [comment]
tested and this integration in working now

**Comment by zmalik** [comment]
@joejulian `0.10.0` isn't released yet. But once it's released I will update this PR once the charts repo is also updated.

**Comment by nfnt** [comment]
There's also the KUDO driver in `kubeaddons` that depends on this addon and needs to be updated as well because it likely won't be compatible. I'll prepare a PR for this.

**Comment by nfnt** [comment]
FYI, with the next bump of KUDO in `kubeaddons`, KUDO will be deployed as part of the `kubeaddons` controller. This will obsolete the `kudo` addon.

See https://github.com/mesosphere/kubeaddons/pull/698 for the bump.

**Comment by zmalik** [comment]
@nfnt this is great! how will it affect UX when deploying kubeaddons? 
or it will be transparent a user deploying a cluster that uses kubeaddons

**Comment by nfnt** [comment]
It will be transparent, as the driver that takes care of installing the addon is an implementation detail. A user simply keeps deploying an addon, `kubeaddons` takes care of deploying it.

**Comment by zmalik** [comment]
closing the PR, as KUDO no longer will be installed as an Addon 
Once mesosphere/kubeaddons#698 is merged will be part of `kubeaddons`
thanks @nfnt

**Pull request closed without merging**